### ISSUE TITLE
tag links with elm version

### ISSUE THREAD

**Issue by bseng** [isRuslan/awesome-elm#89]
since elm 0.17 change a few things in a big way can we mark or sort the links according to versions? 

easier to grasp whats current.

**Comment by isRuslan** [comment]
Hi @bseng, thanks for suggestion, but I don't see the clear way to add tags in markdown format and support in continuously for every link, for every elm version. If any link is not supported anymore we should remove it.

**Comment by HenryMarshall** [comment]
Is a markdown tag absolutely necessary? I feel that denoting the version in parens like this would be sufficient:

> * [Official tutorial](http://elm-lang.org/docs) (0.18) - General information and in-depth guide with examples.
> * [Elm Tutorial](http://www.elm-tutorial.org/) (0.17) - A tutorial on developing single page web applications with Elm.

Alternatively, we could embed little images which would give the benefit of being color coded.

> * [Official tutorial](http://elm-lang.org/docs) ![0-18](https://cloud.githubusercontent.com/assets/2374554/20945610/ee5a4304-bbd5-11e6-875c-1cc6823805d0.png) - General information and in-depth guide with examples.
> * [Elm Tutorial](http://www.elm-tutorial.org/) ![0-17](https://cloud.githubusercontent.com/assets/2374554/20945609/ee5a2400-bbd5-11e6-9285-cea3746cd915.png) - A tutorial on developing single page web applications with Elm.

**Comment by jwoLondon** [comment]
Just flagging this issue again because with the release of Elm 0.19, the elm version is again a more important consideration.

Could use badges:

* A very old app [![elm version](https://img.shields.io/badge/elm-v0.17-yellow.svg?style=flat-square)](http://elm-lang.org)
* A middle-aged app [![elm version](https://img.shields.io/badge/elm-v0.18-orange.svg?style=flat-square)](http://elm-lang.org)
* A new app [![elm version](https://img.shields.io/badge/elm-v0.19-blue.svg?style=flat-square)](http://elm-lang.org)

**Comment by eboCode** [comment]
If sorting's not easy enough, I'd second jwoLondon's badge idea. Maybe both?

I've worked with Elm and SML for several months, but am finding that a lot of code older than 0.19 can be problematic to learn from, especially when you're somewhat new to Elm.

Although I've learned enough to convert most of my own code to 0.19, I've run into lots of tutorial code I couldn't very easily update to 0.19, let alone to translate it in my head while reading it as part of a tutorial.
